_posts/2020-10-27-gromacs.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Testing GROMACS on ARCHER2 4-cab
+date: 2020-10-27 14:00:00
+author: Julien Sindt EPCC
+categories: [news]
+---
+
+One of the upsides of working on the ARCHER2 CSE team is that,
+sometimes, one finds oneself in the interesting position of being 
+the *only* user on the ARCHER2 four-cabinet system (at least
+the only apparent user -- I'm sure that AdrianJ is lurking 
+on one of the compute nodes somewhere). For context, the ARCHER2
+four-cabinet system is roughly equivalent in CPU-count to the
```

Comment:
"CPU-count" should be "core-count" in this sentence (but kept as CPUs in the brackets)